Debug App/Sources/View Controllers/MerchantHeadlessCheckoutAvailablePaymentMethodsViewController.swift

```diff
@@ -175,8 +175,12 @@ extension MerchantHeadlessCheckoutAvailablePaymentMethodsViewController: UITable
             break
             #endif
         default:
-            redirectManager = try? PrimerHeadlessUniversalCheckout.NativeUIManager(paymentMethodType: paymentMethodType)
-            try? redirectManager?.showPaymentMethod(intent: sessionIntent)
+            do {
+                redirectManager = try PrimerHeadlessUniversalCheckout.NativeUIManager(paymentMethodType: paymentMethodType)
+                try redirectManager?.showPaymentMethod(intent: sessionIntent)
+            } catch {
+                print("\n\nMERCHANT APP\n\(#function)\nerror: \(error)")
```

Comment:
@BorisNikolic do we need to send analytics data, or logs at this point to Kibana?

Reply:
Actually no need, this is the debug app code, the view controller utilising headless implementation as an example.